README.md

```diff
@@ -1,3 +1,5 @@
+DUMMY TEST
```

Comment:
this breaks the linter :sweat_smile: 

I usually do `date > ci && git add ci && git commit -m ":robot: CI was here...:fire:" ci` when doing these empty PRss